### DISCUSSION TITLE
Welcome to Thruk Discussions!

### DISCUSSION THREAD

**Discussion by sni** [sni/thruk#1069] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rebagaines** [comment]
Hello!
I am new to Thruk.  I have set up two Nagios Core instances and want to connect them to Thruk using Livestatus.
Not sure how to install Livestatus to connect the backend in Thruk.  Can someone point me in the right direction, please?

**Reply by sni** [reply]
See https://thruk.org/documentation/install.html#_backend-configuration for starters.

**Comment by rebagaines** [comment]
Looing for instructions on how to install livestatus on Nagios core so that I can connect it to Thurk.

The  link talks about livestatus but not how to install it on Ubuntu 0.04

Thanks
